src/main/java/lotto/Application.java

```diff
@@ -1,7 +1,21 @@
 package lotto;
 
+import lotto.domain.ProfitCalculator;
+import lotto.io.ConsoleLottoOutput;
+import lotto.io.UserLottoInput;
+import lotto.service.PurchaseService;
+import lotto.service.generator.RandomLottoGenerator;
+import lotto.service.generator.RandomNumbersGenerator;
+
 public class Application {
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+        LottoClient lottoClient = new LottoClient(
+                new PurchaseService(new RandomLottoGenerator(new RandomNumbersGenerator())),
+                new UserLottoInput(),
+                new ConsoleLottoOutput(),
```

Comment:
```suggestion
                new ConsoleLottoInput(),
                new ConsoleLottoOutput(),
```
이런 식으로 이름 짓는 규칙이 일관되면 좋을 것 같아요.